{"name": "test_get_user", "status": "failed", "statusDetails": {"message": "AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nassert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "user_page = <pages.user_page.UserPage object at 0x000002C089F2B9E0>\n\n    def test_get_user(user_page):\n        with allure.step(\"Getting the created user\"):\n            response = user_page.get_user(\"testuser\")\n>           assert response.status_code == 200, response.text\nE           AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nE           assert 404 == 200\nE            +  where 404 = <Response [404]>.status_code\n\ntest_user_operations.py:37: AssertionError"}, "steps": [{"name": "Getting the created user", "status": "failed", "statusDetails": {"message": "AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nassert 404 == 200\n +  where 404 = <Response [404]>.status_code\n", "trace": "  File \"C:\\Users\\bbuld\\PycharmProjects\\PetShop\\tests\\test_user_operations.py\", line 37, in test_get_user\n    assert response.status_code == 200, response.text\n"}, "start": 1719519606084, "stop": 1719519607200}], "start": 1719519606084, "stop": 1719519607201, "uuid": "3fdacae9-ed7a-4510-9b41-835fb2e84975", "historyId": "5fb4ff886072156a53a7dd3454f2c68c", "testCaseId": "5fb4ff886072156a53a7dd3454f2c68c", "fullName": "tests.test_user_operations#test_get_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "14628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}